tests/index-test.js

```diff
@@ -0,0 +1,168 @@
+'use strict';
+
+const BroccoliTestHelper = require('broccoli-test-helper');
+const buildOutput = BroccoliTestHelper.buildOutput;
+const createTempDir = BroccoliTestHelper.createTempDir;
+const co = require('co');
+const oneLineTrim = require('common-tags').oneLineTrim;
+const stripIndent = require('common-tags').stripIndent;
+
+const CSSReader = require('../index.js').CSSReader;
+
+const addon = require('../index.js');
+
+const test = QUnit.test;
+
+QUnit.module('CSSReader', function(hooks) {
+  let input;
+
+  hooks.beforeEach(co.wrap(function* () {
+    input = yield createTempDir();
+  }));
+
+  hooks.afterEach(co.wrap(function* () {
+    yield input.dispose();
+  }));
+
+  test('concats vendor.css / app.css into style tag inside index.html', co.wrap(function* (assert) {
```

Comment:
I think CSSReader tests are in the other module? My impression was this module was for full integration testing.